### ISSUE TITLE
Syntax highlighting no longer exists

### ISSUE THREAD

**Issue by domainzero** [BookStackApp/BookStack#404]
* BookStack Version: BookStack v0.16.2 (MASTER)
* PHP Version: PHP 5.6.30 (cli)
* MySQL Version: mysql  Ver 15.1 Distrib 10.1.21-MariaDB

##### Expected Behavior
Syntax highlighting should be working.
I pulled from master because I was on auto-pilot, wondering if it broke there.

##### Actual Behavior
No syntax highlighting is present. Code blocks are displayed correctly but I'm not sure if hljs is actually loading. I have an hljs stylesheet loaded into the head of the page and that isn't changing any of the code blocks either.

**Comment by domainzero** [comment]
I'm a doofus and should have read the commits. 
e5fc6bf5faffaa5502155345436469d6c2050510 - Moved from highlight.js to codemirror

I'll see myself out.

**Comment by domainzero** [comment]
Ehhh, I guess I might have closed this early. Syntax highlighting still isn't working correctly.

**Comment by shatteredsword** [comment]
To clarify, you are talking about the syntax highlighting in the source code view of TinyMCE editor, correct?
I've noticed this was missing as well.

**Comment by domainzero** [comment]
I should have been more clear. I think the only reason I hit this issue was because I pulled from the master branch instead of the release branch.

I wasn't getting any syntax highlighting on saved pages after the removal of hljs. For now I think this is an expected issue as codemirror wasn't 100% implemented at the time that I updated.

I'ma close this and we can sweat the issue if it persists when codemirror is fully implemented.
